Tests/JollyTests/CacheTests.swift

```diff
@@ -0,0 +1,103 @@
+import XCTest
+@testable import Jolly
+
+class CacheTests: XCTestCase {
+    
+    var cache: Cache!
+    
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        let defaults = InMemoryUserDefaults()
+        cache = Cache(defaults: defaults)
+    }
+    
+    func testAddRepo() {
+        let repo = Repo(fullName: "inaka/jolly")!
+        
+        cache.add(repo, toRoomWithId: "1")
+        
+        let savedRepos = cache.repos(forRoomWithId: "1")
+        XCTAssertEqual(savedRepos.count, 1)
+        XCTAssertEqual(savedRepos[0].fullName, "inaka/jolly")
+    }
+    
+    func testRemoveExistentRepo() {
+        let repo1 = Repo(fullName: "inaka/jolly")!
+        let repo2 = Repo(fullName: "inaka/jayme")!
+
+        cache.add(repo1, toRoomWithId: "1")
+        cache.add(repo2, toRoomWithId: "1")
+        cache.remove(repo1, fromRoomWithId: "1")
+
+        let savedRepos = cache.repos(forRoomWithId: "1")
+        XCTAssertEqual(savedRepos.count, 1)
+        XCTAssertEqual(savedRepos[0].fullName, "inaka/jayme")
+    }
+    
+    func testRemoveNonExistentRepo() {
+        let repo = Repo(fullName: "inaka/jolly")!
+
+        cache.remove(repo, fromRoomWithId: "1")
+        
+        let savedRepos = cache.repos(forRoomWithId: "1")
+        XCTAssertEqual(savedRepos.count, 0)
+    }
+    
+    func testReposAreSortedAlphabetically() {
+        let repo1 = Repo(fullName: "inaka/jolly")!
+        let repo2 = Repo(fullName: "inaka/jayme")!
+        let repo3 = Repo(fullName: "inaka/galgo")!
+
+        cache.add(repo1, toRoomWithId: "1")
+        cache.add(repo2, toRoomWithId: "1")
+        cache.add(repo3, toRoomWithId: "1")
+        
+        let savedRepos = cache.repos(forRoomWithId: "1")
+        XCTAssertEqual(savedRepos.count, 3)
+        XCTAssertEqual(savedRepos[0].fullName, "inaka/galgo")
```

Comment:
You need these repos to actually exist or you are just using their names as examples, here?